docs/topics/multiplatform/multiplatform-ios-integration-overview.md

```diff
@@ -0,0 +1,67 @@
+[//]: # (title: iOS integration methods)
+
+You can integrate a Kotlin Multiplatform shared module into your iOS app. For that, an [iOS framework](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/WhatAreFrameworks.html)
+is generated from the shared module. Then, you should add it as a dependency to the iOS project:
+
+![iOS integration scheme](ios-integration-scheme.svg)
+
+It's possible to consume this framework as a local or remote dependency. Choose local integration if you want to have
+full control over the entire codebase and get instant updates to final applications when the common code changes.
+
+If you want to explicitly separate the code base of your final application from the common code base, set up remote
+integration. In this case, the shared code will be integrated into final applications like a regular third-party
+dependency.
+
+## Local integration
+
+In a local setup, there are two main integration options. You can use direct integration through a special script or, if
+you have Pod dependencies in your Kotlin Multiplatform project, take the CocoaPods integration approach.
+
+### Direct integration
+
+You can connect the iOS framework directly from the Kotlin Multiplatform project by adding a special script to your Xcode
+project. The `embedAndSignAppleFrameworkForXcode` script is integrated into the build phase of your project's build settings.
+
+This integration method works for you if:
+
+* You do **not** import CocoaPods dependencies in your Kotlin Multiplatform project
```

Comment:
A list with a single item looks a bit strange :) can we combine this into one sentence? I understand the desire to keep the formatting uniform, but not sure the trade-off here is worth it.